examples/gaussian.py

```diff
@@ -15,6 +15,7 @@
 model = A*exp(-((x - x0)**2/(2 * sig**2)))
 
 # Sample 10000 points from a N(15.0, 1.5) distrubution
+np.random.seed(seed=123456789)
```

Comment:
I get the feeling that specifying a seed for an example is a bit strange. I would prefer setting a better initial guess --- this way it seems you can only use symfit if you seed the random generator.